sheduling/src/utils/api.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const host = "http://localhost:3000/login";
+const API = {
+  login: (phone, pass, success) => {
+    axios.post(host, { phone: phone, password: pass }).then(res => {
+      success(res);
```

Comment:
Ensure the `token` get's to the store since you are using the store to access it. You can dispatch an action to the store with the payload containing the token from here, once you receive it from the back-end.